schema/schema2025-08-22.yaml

```
@@ -0,0 +1,1152 @@
+  description: 'RESTful implementation of a light hearted Roast Insult API '
+  contact:
+    name: Terry A. Brooks, Jr.
+    url: https://brooksjr.com
+    email: terry@brooksjr.com
+paths:
+  /api/insult:
+    get:
+      operationId: get_random_insult
+      description: Get a random insult.
+      parameters:
+      - in: query
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)